infra/packaging/build

```diff
@@ -85,7 +85,7 @@ function join_by
 # Invoke "preset_configure" function that the preset provides
 preset_configure
 
-NPROC=$(cat /proc/cpuinfo | grep -c processor)
+NPROC=${NPROC:-$(cat /proc/cpuinfo | grep -c processor)}
```

Comment:
Oh... is this working ;-)? It would be better to add "echo" below to check whether this works as expected.

Reply:
[Reference](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion)
```
${parameter:-word}

If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.
```
@parjong You mean adding "echo" to real PR or just for test?

Reply:
It would be better to keep that change in the actual PR as we did for preset :)

Reply:
As a user, I would like to see whether my configuration is applied as expected